### PR TITLE
New Snippet

### PR DESCRIPTION
**Pull request by stefannypanameno93**
Adding a snippet that calculates if a number is even or odd

### CODE REVIEW

**Review by DeekshithRajBasa — APPROVED** [review]

File: snippets/currentDate.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Ask your name and tells you the current date
+tags: browser, begginer
+---
+
+Ask a user to enter the name and returns current date
```
Comment (DeekshithRajBasa) [inline snippets/currentDate.md:6]: - Declare n1 variable to save the user's name	 
  
 - Declare dayToday to save the current date	 
  
 - Use new Date() to get the current date and hour	 
  
- Declare a function to return the user's name and current date

**Review by DeekshithRajBasa — CHANGES_REQUESTED** [review]
Please check the grammar, thanks.

File: snippets/evenOddNumber.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- Declare evenNumber function to make the division and return the result
+- Declare the variable result to save the result
+- The user sees the result through an alert
+
+```js
+
+var num = prompt("Enter a whole number");
+
+const evenNumber = (num) => {
+  if(num % 2 == 0) {
```
Comment (DeekshithRajBasa) [inline snippets/evenOddNumber.md:18]: return num % 2 == 0 ? "even Number": "odd number";

**Review by DeekshithRajBasa — CHANGES_REQUESTED** [review]
try to reduce the lines of code by using ternary operator

**Review by Trinityyi — COMMENTED** [review]
Thanks for contributing. However, your PR doesn't follow our [contribution guidelines](../blob/master/CONTRIBUTING.md) correctly and it doesn't provide enough value for the average reader. Closing due to that.

**Pull request closed without merging**